docs/source/test_network.md

```diff
@@ -319,7 +319,7 @@ Chaincodes are invoked when a network member wants to transfer or change an
 asset on the ledger. Use the following command to change the owner of a car on
 the ledger by invoking the fabcar chaincode:
 ```
-peer chaincode invoke -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --tls true --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem -C mychannel -n fabcar --peerAddresses localhost:7051 --tlsRootCertFiles ${PWD}/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt --peerAddresses localhost:9051 --tlsRootCertFiles ${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt -c '{"function":"changeCarOwner","Args":["CAR9","Dave"]}'
+peer chaincode invoke -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --tls true --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem -C mychannel -n fabcar --peerAddresses localhost:7051 --tlsRootCertFiles ${PWD}/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt --peerAddresses localhost:9051 --tlsRootCertFiles ${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt -c '{"function":"changeCarOwner","Args":["CAR009","Dave"]}'
```

Comment:
I think this may only be the case in Java ... maybe we should fix the Java fabcar sample instead?

Reply:
This update matches the output from line 315 so I believe (hope?) it's correct:

```
{"Key":"CAR9", "Record":{"make":"Holden","model":"Barina","colour":"brown","owner":"Shotaro"}}
```

Reply:
I ran the network and confirmed, the java chaincode uses a different index than golang and javascript.
```
 nikhilgupta$ peer chaincode query -C channel2 -n fabcar -c '{"Args":["queryCar","CAR009"]}'
{"owner":"Shotaro","model":"Barina","color":"brown","make":"Holden"}
```
For golang
```
peer chaincode query -C channel2 -n fabcar -c '{"Args":["queryCar","CAR9"]}'
{"owner":"Shotaro","make":"Holden","model":"Barina"}
```

Reply:
> I think this may only be the case in Java ... maybe we should fix the Java fabcar sample instead?

You are right! I tested it with the go code the wrong way. Sorry!

Reply:
Ill put a small note in the doc for java users. I can remove it if we update the chaincode.